src/irmin-pack/unix/file_manager.ml

```diff
@@ -214,6 +224,7 @@ struct
       [%log.debug "reload: opening %s" path];
       Prefix.open_ ~readonly:true ~path
     in
+    let* () = notify_reload_consumers t.prefix_consumers in
```

Comment:
This works! No need to fix it for 3.5, but I can't shake the feeling that the notification should happen two lines below, once the new prefix is available to consumers :P

Reply:
Mmm, yeah that's a good point; it's the more semantically correct spot! Not a big change, so I'll update this. Thanks for the :detective: work.